### ISSUE TITLE
CONVERT DATE inline

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1769]
false positive

```abap
DATA date TYPE d.
DATA time TYPE t.
DATA tz TYPE timezone.
CONVERT DATE date TIME time INTO TIME STAMP DATA(timestamp) TIME ZONE tz.
WRITE / timestamp.
```

fyi @JohannesKonings
